.github/workflows/release.yml

```
@@ -129,7 +132,36 @@ jobs:
         with:
           name: azureauth-${{ github.event.inputs.version }}-${{ matrix.runtime }}
           path: azureauth-${{ github.event.inputs.version }}-${{ matrix.runtime }}
-
+  
+  # Build and sign Linux binaries on Azure DevOps and publish them to GitHub and packages.microsoft.com.
+  linux_build_and_sign:
+    runs-on: ubuntu-latest
+    needs: [validate]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2

```

```suggestion
      uses: actions/checkout@v3
```

Checkouts is now on `v3`